module/Core/src/Core/Entity/AttachableEntityInterface.php

```diff
@@ -10,29 +10,27 @@
  */
 namespace Core\Entity;
 
-use Core\Entity\Collection\AttachedEntitiesCollection;
-
 interface AttachableEntityInterface
```

Comment:
You are right, the name is really misleading. What about _ProvideEntityAttachmentInterface_?